src/egon/data/airflow/dags/pipeline.py

```diff
@@ -316,12 +321,14 @@
     zensus_misc_import >> import_district_heating_areas
 
     # Calculate dynamic line rating for HV trans lines
-    calculate_dlr = PythonOperator(
-        task_id="calculate_dlr", python_callable=dlr.Calculate_DLR
-    )
-    osmtgmod_pypsa >> calculate_dlr
-    download_data_bundle >> calculate_dlr
-    download_weather_data >> calculate_dlr
+    dlr = Calculate_dlr(
+        dependencies=[osmtgmod_pypsa,
+                      download_data_bundle,
+                      download_weather_data,
+            ]
+    )
+    #download_data_bundle = tasks["data_bundle.download"]
```

Comment:
Is this still needed? If not, please remove.

Reply:
Removed.